datapipelines/src/main/kotlin/io/customer/datapipelines/config/ScreenView.kt

```diff
@@ -0,0 +1,28 @@
+package io.customer.datapipelines.config
+
+/**
+ * Enum class to define how CustomerIO SDK should handle screen view events.
+ */
+enum class ScreenView {
```

Comment:
Does this mean it's a mutually exclusive choice? I can either choose analytics or in-app? But what if a customer needs both?